db/migrate/20240606012618_remove_auth_from_users.rb

```diff
@@ -4,7 +4,7 @@ def up
       credential = user.credentials.build(
         type: "PasswordCredential",
         password_digest: user.password_digest,
-        last_authenticated_at: user.person.updated_at
```

Comment:
This ran into an error with my dev db, a user witout a person record, so added the rescue-nil

Reply:
Hmm, that's interesting. I don't think it should be possible to have a User without a Person. If that's happening, it suggests a deeper bug in the code and possibly a validation that's missing.

I just did a big extension of auth and credentials (merged into main last week) and some of the auth-related stuff is associated with the user (like credentials) but credentials let you auth different clients (i.e. browsers) and clients are associated with the person.

Anyway, just sharing that as food for thought. It's fine to add the rescue nil in this migration anyway but I think there's a deeper problem somewhere.

Reply:
Maybe my fixtures were not in good shape